mcs/jay/main.c

```diff
@@ -103,12 +103,17 @@ short **derives;
 char *nullable;
 
 #if defined(_WIN32) && !defined(__CYGWIN32__) && !defined(__CYGWIN__)
-extern char* mktemp();
-#define mkstemp mktemp
+#define unlink _unlink
```

Comment:
Does Cygwin ever imply Win32?
i.e. shouldn't this just be `#if _WIN32`?

Reply:
Tried a couple of variant's but it seems like the way this get's build through our default make files, we even get defines like _MCS_VER set in that scenario even though we compile using gcc, so I didn't continue experimenting with it and kept the condition based on how it always been defined.